docker/linksets/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+source ../../common/common.sh
+
+docker-compose up -d  
```

Comment:
While I agree `docker-compose up -d ` should do the job, I find that more often than not, especially after git pulls, there is always a need to rebuild the images. I run this `docker-compose up -d --build --force-recreate`. It only takes a tiny bit more time to run, and in the context of time running the whole container its nothing, but it ensures the latest code is always running.

Reply:
Yes this is better, I've changed the code . I've used patterns like `docker-compose build --no-cache && docker-compose up -d` previously because cache change detection in docker is limited and --no-cache ensures a complete rebuild. There also the possibility to remove the use the shell wrapper all together and try as much as possible.